controller/space.go

```diff
@@ -325,6 +330,9 @@ func ConvertSpace(request *goa.RequestData, p *space.Space, additional ...SpaceC
 		},
 		Links: &app.GenericLinks{
```

Comment:
Do you mean I remove `Meta` and do like this - https://github.com/almighty/almighty-core/pull/1016/files#diff-a91093f84923c689d4e35b9be6190521R21 ?